OpenRA.m

```diff
@@ -142,6 +147,13 @@ - (void)launchGameWithArgs: (NSArray *)gameArgs
 	[launchArgs addObject: @"--debug"];
 	[launchArgs addObject: [gamePath stringByAppendingPathComponent: gameName]];
 	[launchArgs addObject: [NSString stringWithFormat:@"Engine.LaunchPath=\"%@\"", appPath]];
+
+	if (defaultModId)
+	{
+		[launchArgs addObject: [NSString stringWithFormat:@"Engine.DefaultMod=%@", defaultModId]];
```

Comment:
We can keep this for now, because it will be needed during the transition and won't do any harm after it is complete.